File: Java/01.Java-Basic/08.WhileLoop/src/whileLoop/lab/SumNumbers.java
package whileLoop.lab;

import java.util.Scanner;

public class SumNumbers {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int number = Integer.parseInt(scanner.nextLine());
        int sum = 0;
        while (number > sum) {
            int a = Integer.parseInt(scanner.nextLine());
            sum += a;
        }

        System.out.println(sum);
    }
}

File: JavaScript/01.JavaScriptBasic/09While-Loop-Lab/src/sumNumbers.js
function sum(params) {
    let number = Number(params[0]);
    let total = 0;
    let index = 1;
    let n = Number(params[index]);
    while (total < number) {
        total += n;
        index++;
        n = Number(params[index]);
    }

    console.log(total);
}

sum(["100",
"10",
"20",
"30",
"40"]);